docs/ref/models/expressions.txt

```diff
@@ -450,6 +450,116 @@ Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
 :keyword:`else` logic in queries. Django natively supports SQL ``CASE``
 expressions. For more details see :doc:`conditional-expressions`.
 
+``Subquery()`` expressions
+--------------------------
+
+.. currentmodule:: django.db.models.expressions
+
+.. class:: Subquery(queryset, output_field=None)
+
+.. versionadded:: 1.11
+
+When a value is required that is dependent upon values from within a related
+object, it's possible to use an explicit ``Subquery``. For example::
```

Comment:
Yeah, I think maybe I'll rewrite the examples to use an existing model structure, or at least something with "known" semantics. Perhaps User, Post, Comment?